### PR TITLE
T/334: Support for BrowserStack

### LINKED ISSUE #334: Support for BrowserStack
In order to run tests on BrowserStack (https://github.com/ckeditor/ckeditor5/issues/615), we need to adjust our test tools.

### PR DESCRIPTION
**Pull request by pomek**
### Suggested merge commit message ([convention](https://github.com/ckeditor/ckeditor5-design/wiki/Git-commit-message-convention))

Feature: Tests can be executed using BrowserStack. Closes #334.

NOTE: In order to execute tests using BrowserStack, you need to call `BROWSER_STACK_USERNAME=[...] BROWSER_STACK_ACCESS_KEY=[...] npm test -- --browsers=[...]`.

### CODE REVIEW

File: packages/ckeditor5-dev-tests/README.md [hunk 1]
```diff
@@ -50,9 +50,23 @@ You can also use the bin script for testing a package:
...
+
+You can use BrowserStack to execute the tests on different browsers. At this moment we have defined 4 browsers:
+
+1. Edge 16 - Windows 10
+1. Chrome 62 - OS X Mavericks
+1. Firefox 6757 - OS X Yosemite
+1. Safari 11 - OS X High Sierra
+
+In order to run BrowserStack, you need to specify the `--username` and `--access-key` options. By default, we
+execute the tests on all browsers listed above. You can use an option `--browser` to change it.
```
Comment (ma2ciek) [inline packages/ckeditor5-dev-tests/README.md]: `browser` or `browsers`?

Comment (pomek) [reply packages/ckeditor5-dev-tests/README.md]: `browsers` ofc.

File: packages/ckeditor5-dev-tests/README.md [hunk 2]
```diff
@@ -87,6 +101,18 @@ $ npm t -- --files='!(engine)'
...
+Test the `ckeditor5-engine` using BrowserStack and all available browsers:
+
+```bash
+$ npm test -- -username [...] -a [...] --files engine
+```
+
+Test the `ckeditor5-image` and `ckeditor5-ui` using BrowserStack on Safari and Edge:
+
+```bash
+$ npm test -- -username [...] -a [...] --files image,ui --browser=safari,edge
```
Comment (ma2ciek) [inline packages/ckeditor5-dev-tests/README.md]: Also, `browser` -> `browsers`.

**Comment by pomek** [comment]
First build using BrowserStack – https://travis-ci.org/ckeditor/ckeditor5-autoformat/builds/319050372#L2288.

image

**Comment by pomek** [comment]
On the previous screen, Travis had Firefox 56.

I upgraded it and Travis uses FF 57 now.

image

**Pull request merged**